components/resources/brave_components_strings.grd

```diff
@@ -255,6 +255,18 @@
       <message name="IDS_BRAVE_NEW_TAB_SEARCH_HIDE_SEARCH" desc="Message for the menu entry to hide the search box on the NTP">
         Hide Search
       </message>
+      <message name="IDS_BRAVE_NEW_TAB_SEARCH_ENABLE_SEARCH_SUGGESTIONS_TITLE" desc="Title for the Enable search suggestions prompt">
+        Enable search suggestions?
+      </message>
+      <message name="IDS_BRAVE_NEW_TAB_SEARCH_ENABLE_SEARCH_SUGGESTIONS_EXPLANATION" desc="Explanation for the Enable search suggestions prompt">
+        When you start a search in what you type will be sent to your search engine for better suggestions.
```

Comment:
Looks like some of the text is missing: ```When you start a search in your browser's address bar, what you type will be sent to your default search engine for better suggestions.```

Reply:
I left a comment on the design explaining why this is like it is:
1. This isn't for the browsers address bar only (it's on the NTP too) - this displays on the NTP, so its particularly confusing.
2. Its not only for the default search engine (if you change to using DDG or Google on the NTP we'll send your keystrokes to them).

Reply:
https://www.figma.com/design/lVTufBkl5eNrtOQjlAHxjb?node-id=1334-23612#849358459

Reply:
oh ok, that makes sense. I think we might want to keep the `,` though - makes it a bit easier to read, imo.

Reply:
Done :smile: